frontend/README.md

```
@@ -1,28 +1,24 @@
-## 🖥️ Frontend
-
+🖥️ Frontend
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)